MeNext/Layout/PlayingScreen.cs

```diff
@@ -26,17 +26,14 @@ public PlayingScreen(MainController mainController)
                 Command = new Command(() => mainController.RequestPrevious())
             });
 
+            var playCommand = new Command(() =>
+            {
+                mainController.musicService.Playing = !mainController.musicService.Playing;
+            });
             layout.Children.Add(new Button
             {
                 Text = "Play/Pause",
-                Command = new Command((obj) =>
-                {
-                    if (mainController.musicService.Playing) {
-                        mainController.RequestPause();
-                    } else {
-                        mainController.RequestPlay();
-                    }
-                })
```

Comment:
Please undo this change. We should be requesting a play/pause from the server and then the server gives us a new status. What you switched it to only toggles play/pause if the person is the host, and doesn't let the clients know what's happened.

@ConnorFoody Is play/pause implemented server-side yet? Who should write the code for interfacing with the backend api? Who do we want to rewrite PlayController to make it work with play/pause?